src/app/homepage/homepage.module.ts

```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+
+import { HomepageComponent } from './homepage.component';
+import {HomepageRoutingModule} from "./homepage-routing.module";
```

Comment:
If you run tslint it shows an error here. Please configure your IDE to show lint errors